packages/core/parcel-bundler/src/scope-hoisting/shake.js

```diff
@@ -122,6 +114,15 @@ function remove(path) {
   } else if (isUnusedWildcard(path)) {
     remove(path.parentPath);
   } else if (!path.removed) {
-    path.remove();
+    if (
```

Comment:
The code duplication isn't ideal but I don't see a cleaner way without doing some extra if checks